backends/apple/coreml/partition/coreml_partitioner.py

```diff
@@ -89,6 +91,8 @@ def partition(self, exported_program: ExportedProgram) -> PartitionResult:
                 partition_tags[tag] = self.delegation_spec
 
         tag_constant_data(exported_program)
+        if self.take_over_mutable_buffer:
```

Comment:
Also maybe add a log here if use enables mutable buffer, as they may not aware the OS requirements

Reply:
Added log info as suggested